src/compas/scene/sceneobject.py

```diff
@@ -176,21 +169,17 @@ def transformation(self, transformation):
     @property
     def worldtransformation(self):
         # type: () -> compas.geometry.Transformation
-        frame_stack = [self.frame] if self.frame else []
-        parent = self.parent
+        transformations = [self.transformation] if self.transformation else []
+        parent: SceneObject = self.parent
```

Comment:
i like your enthusiasm, but this annotation can't be introduced yet in  the current version of COMPAS...

Reply:
removed